src/components/Layout/DrawerNavigation.tsx

```diff
@@ -118,18 +118,30 @@ export const DrawerNavigationComponent: FunctionComponent<
   const onPressCloseDrawer = (): void => {
     navigation.dispatch(DrawerActions.closeDrawer());
   };
-  const releaseChannel: string | undefined = Constants.manifest?.releaseChannel;
+  const branchName: string | undefined = (Constants.manifest2?.metadata as any)
+    ?.branchName
+    ? Constants.manifest?.releaseChannel
+    : (Constants.manifest2?.metadata as any)?.branchName;
+  const manifestVersion = Constants.manifest?.version
+    ? Constants.manifest?.version
+    : Constants.manifest2?.extra?.expoClient?.version;
   let version = "";
-  if (releaseChannel) {
-    version += `ver ${Constants.manifest?.version}`;
-    if (Constants.manifest?.extra?.appBuildVersion) {
-      version += ` / ${Constants.manifest.extra.appBuildVersion}`;
+  const appBuildVersion = Constants.manifest?.extra?.appBuildVersion
+    ? Constants.manifest?.extra?.appBuildVersion
+    : Constants.manifest2?.extra?.expoClient?.extra?.appBuildVersion;
+  if (__DEV__) {
```

Comment:
not doing if (branchName) because branchName isnt guaranteed to exist as `Constants.manifest2?.metadata` doesnt have a type